app/api/deleteFile/route.ts

```diff
@@ -0,0 +1,53 @@
+import { google } from 'googleapis';
+import { NextRequest, NextResponse } from 'next/server';
+
+export async function DELETE(req: NextRequest) {
+  const { searchParams } = new URL(req.url);
+  const fileId = searchParams.get('fileId');
+
+  if (!fileId) {
+    return NextResponse.json(
+      { message: 'File ID is required' },
+      { status: 400 }
+    );
+  }
+  try {
+    const private_key = process.env.GDRIVE_PRIVATE_KEY?.replace(/\\n/g, '\n');
+
+    if (!private_key) {
+      return NextResponse.json(
+        { error: 'Missing Google Drive private key' },
+        { status: 500 }
+      );
+    }
+
+    const auth = new google.auth.GoogleAuth({
+      projectId: process.env.GDRIVE_PROJECT_ID,
+      scopes: 'https://www.googleapis.com/auth/drive',
+      credentials: {
+        type: 'service_account',
+        client_id: process.env.GDRIVE_PROJECT_ID,
```

Comment:
Shouldn't this be the process.env.GDRIVE_CLIENT_ID? or is that not needed?

Reply:
Not needed, deleted and yeeted.